interface-definitions/load-balancing-haproxy.xml.in

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="load-balancing">
+    <children>
+      <node name="reverse-proxy" owner="${vyos_conf_scripts_dir}/load-balancing-haproxy.py">
+        <properties>
+          <help>Configure reverse-proxy load-balancing</help>
```

Comment:
IMHO `load-balancing` is redundant here.

Reply:
Done!